cranelift/codegen/src/isa/aarch64/inst/args.rs

```diff
@@ -764,3 +769,15 @@ impl APIKey {
         0xd503201f | (crm << 8) | (op2 << 5)
     }
 }
+
+pub use crate::isa::aarch64::lower::isle::generated_code::TestBranchKind;
+
+impl TestBranchKind {
+    /// Inverts this branch condition.
+    pub fn invert(&self) -> TestBranchKind {
```

Comment:
Can we call this `complement`? That's what we call the `IntCC` version after a bunch of bike shedding names.